jetlinks-components/gateway-component/src/main/java/org/jetlinks/community/gateway/supports/DeviceGatewayProvider.java

```diff
@@ -4,6 +4,11 @@
 import org.jetlinks.community.network.NetworkType;
 import reactor.core.publisher.Mono;
 
+/**
```

Comment:
```java
/**
 * 设备网关支持提供商,用于提供对各种设备网关的支持.在启动设备网关时,会根据对应的提供商以及配置来创建设备网关.
 * 实现统一管理网关配置,动态创建设备网关.
 *
 * @author zhouhao
 * @see DeviceGateway
 * @since 1.0
 */
```
参照类似格式,提供这个类的用途,说明,参照类等信息。